contract_queue_job/data/ir_config_parameter.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo noupdate="1">
+    <record id="config_param_contract_queue_job_as_job" model="ir.config_parameter">
+        <field name="key">contract.queue.job.as.job</field>
```

Comment:
I think "contract.queue.job.as.job" is a little bit confusing, "contract.queue.job" should be clear enough?